src/Servers/IIS/build/assets.props

```diff
@@ -1,6 +1,7 @@
 <!-- This file is autogenerated -->
 <Project>
     <PropertyGroup>
+        <BuildIisNativeProjects Condition="'$(SolutionName)' == 'IISIntegration'">true</BuildIisNativeProjects>
```

Comment:
Which doesn't need to check we're running in VS because SolutionName is only ever set in VS?

Reply:
Yeah